.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!---
+BEFORE YOU SUBMIT please search open/closed issues since someone might have asked the same thing before!
+-->
+
+### Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
```

Comment:
In the future a bot could rip out this information to automatically add github labels